rust/private/expand.bzl

```diff
@@ -0,0 +1,247 @@
+# Copyright 2020 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A module defining Rust expansion rules"""
```

Comment:
Could you expand this module header a bit? I think it'd be nice to have some of the context currently in `examples/expand/README.md` here so we could render the docs for folks with the rest of rules_rust docs.